### DISCUSSION TITLE
ChatPDF Support

### DISCUSSION THREAD

**Discussion by Legitimaximus** [n3d1117/chatgpt-telegram-bot#389] [Ideas]
It would be incredible to have ChatPDF support built in. This would require a paid subscription for the user, but $5 a month for 500 messages and 5000 pages isn't too bad. Link to API docs included below:

https://www.chatpdf.com/docs/api/backend

**Comment by noriellecruz** [comment]
no. this functionality could be easily done with python.

no need to subscribe to bs services

**Reply by Legitimaximus** [reply]
Even better then. That's just admittedly beyond me, no problem admitting my amateurism.
